pygmt/__init__.py

```diff
@@ -95,7 +95,11 @@ def _get_ghostscript_version():
         Get ghostscript version.
         """
         os_name = sys.platform
-        if os_name.startswith("linux") or os_name == "darwin":
+        if (
+            os_name.startswith("linux")
+            or os_name.startswith("freebsd")
+            or os_name == "darwin"
+        ):
```

Comment:
```suggestion
        if os_name.startswith(("linux", "freebsd", "darwin")):
```
Same here, I think we can simplify it like so?